tools/lotus2/lotus2.xml

```diff
@@ -106,24 +109,30 @@ lotus2
 #end if
 
 -clustering $clu_args.clu_cond.clustering
-#if $clu_args.clu_cond.clustering in ('1', '3'):
-    -id $clu_args.clu_cond.id
+#if ($clu_args.clu_cond.clustering in ('1', '3') ) and ( $clu_args.clu_cond.id != 0.97 ):
```

Comment:
What difference does this change make?

Reply:
doesn't give the value to lotus2. You have to avoid giving command args to lotus where possible.. there is an intricate system of choosing best default values, based on other parameters, and to take advantage of this system (and not to replicate it again in galaxy), you need to not give flags to the lotus2 call

Reply:
Please don't do that, it's an unexpected behaviour for the user to see a value that is then not passed to the program.
For integer/float parameters, the right solution is to make the input optional and leave the value empty, see suggestions. Note that the easiest way to check in Cheetah if an optional integer/float parameter has been set by the user is to cast it to str.
```suggestion
#if $clu_args.clu_cond.clustering in ('1', '3') and str($clu_args.clu_cond.id):
```